src/js/renderers/ArrayControlRenderer.js

```diff
@@ -25,14 +24,21 @@ import { NavigationButtonAsItem } from './NavigatorLayout/navigation-button';
 import {
     __experimentalUseNavigator as useNavigator,
 	__experimentalHStack as HStack,
-    __experimentalItemGroup as ItemGroup,
-	__experimentalItem as Item,
 	FlexItem,
 } from '@wordpress/components';
 
-export const GutenbergArrayRenderer = (props) => {
+export const GutenbergArrayRenderer = (ownControlProps) => {
+    const ctx = useJsonForms();
```

Comment:
Follow this
https://github.com/eclipsesource/jsonforms/blob/d6f1fda00cdf9502c144b3706ebc7f555a8c6fee/packages/react/src/JsonFormsContext.tsx#L574